tests/spectron/notificationPosition.spectron.js

```diff
@@ -45,9 +45,9 @@ describe('Tests for Notification position', () => {
     it('should load demo html page', () => {
         let filePath;
         if (process.platform === 'win32') {
-            filePath = 'file:///' + path.join(__dirname, '..', 'demo/index.html');
+            filePath = 'file:///' + path.join(__dirname, '..', '..', 'demo/index.html');
```

Comment:
Can we just do `path.join(__dirname, '../../', 'demo/index.html'? Won't that work?